lib/stytch/b2b_discovery.rb

```diff
@@ -21,18 +27,80 @@ def initialize(connection)
+      #
+      #   If a `session_token` or `session_jwt` is provided then a successful authentication will continue to extend the session this many minutes.
+      #
+      #   If the `session_duration_minutes` parameter is not specified, a Stytch session will be created with a 60 minute duration. If you don't want
+      #   to use the Stytch session product, you can ignore the session fields in the response.
+      #   The type of this field is nilable `Integer`.
+      # session_custom_claims::
+      #   Add a custom claims map to the Session being authenticated. Claims are only created if a Session is initialized by providing a value in
+      #   `session_duration_minutes`. Claims will be included on the Session object and in the JWT. To update a key in an existing Session, supply a new value. To
+      #   delete a key, supply a null value. Custom claims made with reserved claims (`iss`, `sub`, `aud`, `exp`, `nbf`, `iat`, `jti`) will be ignored.
+      #   Total custom claims size cannot exceed four kilobytes.
+      #   The type of this field is nilable `object`.
+      # locale::
+      #   (no documentation yet)
+      #   The type of this field is nilable `ExchangeRequestLocale`.
+      #
+      # == Returns:
+      # An object with the following fields:
+      # request_id::
+      #   Globally unique UUID that is returned with every API call. This value is important to log for debugging purposes; we may ask for this value to help identify a specific API call when helping you debug an issue.
+      #   The type of this field is `String`.
+      # member_id::
+      #   Globally unique UUID that identifies a specific Member.
+      #   The type of this field is `String`.
+      # session_token::
+      #   A secret token for a given Stytch Session.
+      #   The type of this field is `String`.
+      # session_jwt::
+      #   The JSON Web Token (JWT) for a given Stytch Session.
+      #   The type of this field is `String`.
+      # member::
+      #   The [Member object](https://stytch.com/docs/b2b/api/member-object).
+      #   The type of this field is `Member`.
+      # organization::
+      #   The [Organization object](https://stytch.com/docs/b2b/api/organization-object).
+      #   The type of this field is `Organization`.
+      # status_code::
+      #   The HTTP status code of the response. Stytch follows standard HTTP response status code patterns, e.g. 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX are server errors.
+      #   The type of this field is `Integer`.
+      # member_session::
+      #   The [Session object](https://stytch.com/docs/b2b/api/session-object).
+      #   The type of this field is nilable `MemberSession`.
       def exchange(
         intermediate_session_token:,
         organization_id:,
         session_duration_minutes: nil,
-        session_custom_claims: nil
+        session_custom_claims: nil,
+        locale: nil
```

Comment:
whoops, `locale` shouldn't be included yet (part of MFA). I'll update in the proto repo

Reply:
merged the [proto change](https://github.com/stytchauth/protobuf/commit/dae38183e8871f863cdab46f948618f2ca797532)